### ISSUE TITLE
downloaded zip file names not unique

### ISSUE THREAD

**Issue by thimblebird** [PurpurMC/PurpurPacks#20]
i'm guessing i can't install more than one datapack (via modrinth at least) because they share file names

**Comment by RhythmicSys** [comment]
Which ones are shared? Can you link me to them? They should be unique

**Comment by RhythmicSys** [comment]
Oh wait I found it, I hadn't realized any of those actually went through, my mistake

**Comment by thimblebird** [comment]
the problem seems to be that the zip files are named `purpurpack_crafting_4.1.zip` for example, which means i can only install either https://modrinth.com/datapack/purpurpack-blasting-smelts-glass or https://modrinth.com/datapack/purpurpack-smelt-raw-ore-blocks

**Comment by RhythmicSys** [comment]
Yes this should be solved now, the issue was caused by the deploy script I was writing, which didn't have the names set up properly initially but I didn't think it actually deployed any of the packs. I've fixed the script now and all the packs should be published for 1.21.4 now

**Comment by thimblebird** [comment]
ah alright, thought i'd spell it out just in case. thanks!

**Comment by thimblebird** [comment]
the commits xD oh no
